Tests/MongoSwiftTests/CodecTests.swift

```diff
@@ -11,8 +11,8 @@ final class CodecTests: MongoSwiftTestCase {
             DecodingError.Context(codingPath: [], debugDescription: "dummy error"))
 
     func testEncodeListDatabasesOptions() throws {
-        let options = ListDatabasesOptions(filter: ["a": 10], nameOnly: true)
-        let expected: Document = ["filter": ["a": 10] as Document, "nameOnly": true]
+        let options = ListDatabasesOptions(filter: ["x": 1], nameOnly: true)
```

Comment:
morbid curiosity: what's going on with this change?

Reply:
So in a previous commit I removed filter from the options, then in a later one I added it back and needed to fix this test. Instead of reverting it I just threw a random document in there. I'll fix this so this diff goes away.